src/site/markdown/server/manage-webadmin.md

```diff
@@ -50,7 +50,6 @@ Finally, please note that in case of a malformed URL the 400 bad request respons
  - [Administrating mail queues](#Administrating_mail_queues)
  - [Administrating DLP Configuration](#Administrating_DLP_Configuration)
  - [Administrating Sieve quotas](#Administrating_Sieve_quotas)
- - [Event Dead Letter](#Event_Dead_Letter)
```

Comment:
Why?

Reply:
Oh, i plained to put it in `## HealthCheck` area right be low this. I believed that suit for this and then i forgot it.

Reply:
This is a link from a table of content to the event dead letter management, we should avoid removing it...

Reply:
Ok, i will add it back.